android/build.gradle

```
@@ -27,6 +27,7 @@ if (isNewArchitectureEnabled()) {
 }
 
 android {
+    namespace = "cl.json"
```

did you check the compatibility with RN < 0.71?
I would suggest something like this 

```suggestion
def agpVersion = com.android.Version.ANDROID_GRADLE_PLUGIN_VERSION.tokenize('.')[0].toInteger()
def shouldUseNameSpace = agpVersion >= 7
def PACKAGE_PROP = "package=\"cl.json\""
def manifestOutFile = file("${projectDir}/src/main/AndroidManifest.xml")
def manifestContent = manifestOutFile.getText()
if(shouldUseNameSpace){
      manifestContent = manifestContent.replaceAll(
        PACKAGE_PROP,
        ''
    )  
} else {
    if(!manifestContent.contains("$PACKAGE_PROP")){
        manifestContent = manifestContent.replace(
            '<manifest',
            "<manifest $PACKAGE_PROP "
        )
    }
}
manifestContent.replaceAll("  ", " ")
manifestOutFile.write(manifestContent)

android {
    if(shouldUseNameSpace){
        namespace = "cl.json"
    }
```

reference: https://github.com/luggit/react-native-config/pull/763/files